### DISCUSSION TITLE
Field Notes

### DISCUSSION THREAD

**Discussion by ic360-com** [ExpressionEngine/ExpressionEngine#3002] [Feature Requests]
Much like the with the Template Notes... sometimes it would be really useful to be able to add Notes to Fields in the Edit Field form... which are invisible from the Edit Entry form... and meant for use only by those with Field editing permissions.
For example:
"In the future, this Select form field could be expanded to include other product discount types, by changing the variable options/values supplied."

[ Y ] I am capable and would like to work on implementation of this feature if it is considered... but am only able to contribute in a design / planning capacity.

**Comment by intoeetive** [comment]
@TomJaeger bringing this into your consideration

**Comment by blue-dreamer** [comment]
This sounds like a really useful feature.

**Comment by intoeetive** [comment]
PR for this feature: https://github.com/ExpressionEngine/ExpressionEngine/pull/3945
